server/routes/requests.js

```diff
@@ -106,18 +99,10 @@ router.patch('/updateRequest/:id/:timeOffset', passport.authenticate('jwt', {ses
             let mentorTime = timezoneHelpers.applyTimezone(request.time, mentor.timeZone)
             let mentorTimeString = `${mentorTime[0].day} ${timezoneHelpers.getSlot(mentorTime[0].time)}`
             let studentsSubscribed = [], alumniSubscribed = []
-            let grade = null
-            let role = null
-            if (request.requesterRole === 'STUDENT') {
-                alumniSubscribed = [mentor._id]
-                studentsSubscribed = [mentee._id]
-                grade = mentee.grade
-                role = 'BOTH'
-            } else {
-                // mentor is first entry in list
```

Comment:
This will need a corresponding change in the `Newsfeed > createCallConfirmedPost`